{
	"alias": "FEDORA-2015-11995",
	"bugs": [
		{
			"bug_id": 1224999,
			"parent": true,
			"security": true,
			"title": "CVE-2013-7440 python: wildcard matching rules do not follow RFC 6125",
			"bugzilla": {
				"bug_id": "1224999",
				"alias": "CVE-2013-7440",
				"creation_ts": "2015-05-26 11:52:59 +0000",
				"short_desc": "CVE-2013-7440 python: wildcard matching rules do not follow RFC 6125",
				"delta_ts": "2021-10-21 00:45:32 +0000",
				"bug_status": "CLOSED",
				"resolution": "ERRATA",
				"keywords": "Security",
				"priority": "low",
				"bug_severity": "low",
				"depends_on": [
					"1023742",
					"1035223",
					"1230951",
					"1230952",
					"1230953",
					"1230954",
					"1231231",
					"1231232",
					"1231238",
					"1231239",
					"1304146",
					"1304225",
					"1304227"
				],
				"blocked": [
					{
						"bug_id": "1225002",
						"error": "NotPermitted"
					}
				],
				"external_bugs": {
					"text": "RHSA-2016:1166",
					"name": "Red Hat Product Errata"
				},
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "8265872",
						"comment_count": "0",
						"who": {
							"text": "mprpic",
							"name": "Martin Prpiƒç"
						},
						"bug_when": "2015-05-26 11:52:59 +0000",
						"thetext": "It was found that Python's SSL hostname matching rules did not conform to RFC 6125 when the hostname included wildcards.\n\nUpstream issue:\n\nhttps://bugs.python.org/issue17997#msg194950\n\nCVE assignment:\n\nhttp://seclists.org/oss-sec/2015/q2/523\n\nUpstream patch:\n\nhttps://hg.python.org/cpython/rev/10d0edadbcdd"
					},
					{
						"isprivate": "0",
						"commentid": "8325478",
						"comment_count": "1",
						"who": {
							"text": "thoger",
							"name": "Tomas Hoger"
						},
						"bug_when": "2015-06-11 08:18:19 +0000",
						"thetext": "This was fixed upstream in Python 3.3.3, and also in backports.ssl_match_hostname 3.4.0.2:\n\nhttps://pypi.python.org/pypi/backports.ssl_match_hostname/3.4.0.2\n\nCorrected version was also added to Python 2.7.9.  Python 2.7 versions before 2.7.9 did not include match_hostname() and hence were not affected."
					},
					{
						"isprivate": "0",
						"commentid": "8328666",
						"comment_count": "3",
						"who": {
							"text": "thoger",
							"name": "Tomas Hoger"
						},
						"bug_when": "2015-06-11 20:55:07 +0000",
						"thetext": "The backports.ssl_match_hostname is bundled with urllib3 upstream sources.  Additionally, urllib3 is bundled in the requests library.  However, python-urllib3 and python-requests packages in Red Hat products and Fedora do not bundle backports.ssl_match_hostname and instead use the version in python-backports-ssl_match_hostname packages.\n\n\nThere are several multiple copies of match_hostname implementation in python-pip:\n\n- pip/backwardcompat/ssl_match_hostname.py\n  - this was used in pip version prior to 1.5, removed via:\n    https://github.com/pypa/pip/commit/c61b5df\n  - version included in pip 1.4.1 was affected by CVE-2013-7440, but not\n    affected by CVE-2013-2099 (bug 963260)\n  - version included in pip 1.3.4 is affected by both CVE-2013-7440 and\n    CVE-2013-2099, but some Red Hat packages already include patch for\n    CVE-2013-2099\n\n- pip/_vendor/requests/packages/urllib3/packages/ssl_match_hostname/_implementation.py\n  - embedded requests library with embedded urllib3 with embedded\n    backports.ssl_match_hostname\n  - added in pip 1.5 when requests 2.0.0 was added, using ssl_match_hostname\n    version with this issue already fixed\n    https://github.com/pypa/pip/commit/651a961\n\n- pip/_vendor/distlib/compat.py\n  - code was affected by both CVE-2013-2099 and CVE-2013-7440 until pip 6.0.0,\n    when embedded distlib was updated to version 0.2.0:\n    https://github.com/pypa/pip/commit/320a07f\n  - it's unclear to me if match_hostname copy in distlib is used by pip\n\n\ndistlib is also packaged separately in Fedora.  The current version of python-distlib packages is 0.1.9.  CVE-2013-2099 and CVE-2013-7440 were only fixed upstream in 0.2.0 via:\nhttps://bitbucket.org/vinay.sajip/distlib/commits/1e44690\n\n\nmatch_hostname code is also copied in the setuptools package.\n\n- 0.7 introduced the code when it add support for SSL certificate verification\n  https://bitbucket.org/pypa/setuptools/commits/8dc5794\n- 0.9.5 corrected CVE-2013-2099\n  https://bitbucket.org/pypa/setuptools/commits/4b0fb61\n- 1.3 corrected CVE-2013-7440\n  https://bitbucket.org/pypa/setuptools/commits/7eeb678\n\npython-setuptools in Red Hat Enterprise Linux 6 and earlier are based on upstream versions prior to 0.7 and hence do not contain match_hostname.  The version in Red Hat Enterprise Linux 7 is 0.9.8.  However, it does not contain match_hostname, as it was patched to use implementation from the python-backports-ssl_match_hostname package.  Version 0.9.8 is also used in python-setuptools packages in python27 and python33 collections in Red Hat Software Collections.  Those packages already include a patch for CVE-2013-7440.  The rh-python34 collections contains newer upstream setuptools versions (11.3.1)."
					},
					{
						"isprivate": "0",
						"commentid": "8328694",
						"comment_count": "4",
						"who": {
							"text": "thoger",
							"name": "Tomas Hoger"
						},
						"bug_when": "2015-06-11 21:08:40 +0000",
						"thetext": "\nCreated python-pip tracking bugs for this issue:\n\nAffects: fedora-all [bug 1230953]\nAffects: epel-all [bug 1230954]"
					},
					{
						"isprivate": "0",
						"commentid": "8328695",
						"comment_count": "5",
						"who": {
							"text": "thoger",
							"name": "Tomas Hoger"
						},
						"bug_when": "2015-06-11 21:08:48 +0000",
						"thetext": "\nCreated bzr tracking bugs for this issue:\n\nAffects: fedora-all [bug 1230951]"
					},
					{
						"isprivate": "0",
						"commentid": "8328696",
						"comment_count": "6",
						"who": {
							"text": "thoger",
							"name": "Tomas Hoger"
						},
						"bug_when": "2015-06-11 21:08:54 +0000",
						"thetext": "\nCreated python-distlib tracking bugs for this issue:\n\nAffects: fedora-all [bug 1230952]"
					},
					{
						"isprivate": "0",
						"commentid": "8330747",
						"comment_count": "7",
						"who": {
							"text": "thoger",
							"name": "Tomas Hoger"
						},
						"bug_when": "2015-06-12 12:29:18 +0000",
						"thetext": "pip and setuptools are bundled with virtualenv.  The following provides a quick overview of versions bundled with virtualenv versions as found in Red Hat products and Fedora:\n\n- virtualenv 1.7.2 - includes distribute-0.6.27 and setuptools-0.6c11 - not affected\n- virtualenv 1.10.1 - includes pip-1.4.1 and setuptools-0.9.8 - affected\n- virtualenv 1.11.6 - includes pip-1.5.6 and setuptools-3.6 - only pip/distlib affected\n- virtualenv 12.0.7 - includes pip-6.0.8 and setuptools-12.0.5 - not affected"
					},
					{
						"isprivate": "0",
						"commentid": "8330795",
						"comment_count": "8",
						"who": {
							"text": "thoger",
							"name": "Tomas Hoger"
						},
						"bug_when": "2015-06-12 12:45:51 +0000",
						"thetext": "match_hostname is also embedded in pymongo:\n\n- it was introduced in 2.5:\n  https://jira.mongodb.org/browse/PYTHON-466\n  https://github.com/mongodb/mongo-python-driver/commit/48046b2e\n- CVE-2013-2099 was corrected in 2.6:\n  https://jira.mongodb.org/browse/PYTHON-522\n  https://github.com/mongodb/mongo-python-driver/commit/e4f6e4f7\n- CVE-2013-7440 was corrected in 2.7\n  https://jira.mongodb.org/browse/PYTHON-650\n  https://github.com/mongodb/mongo-python-driver/commit/9b0e542a"
					},
					{
						"isprivate": "0",
						"commentid": "8330920",
						"comment_count": "9",
						"who": {
							"text": "thoger",
							"name": "Tomas Hoger"
						},
						"bug_when": "2015-06-12 13:13:39 +0000",
						"thetext": "\nCreated python-pymongo tracking bugs for this issue:\n\nAffects: fedora-all [bug 1231231]\nAffects: epel-all [bug 1231232]"
					},
					{
						"isprivate": "0",
						"commentid": "8330979",
						"comment_count": "10",
						"who": {
							"text": "thoger",
							"name": "Tomas Hoger"
						},
						"bug_when": "2015-06-12 13:23:18 +0000",
						"thetext": "\nCreated zeroinstall-injector tracking bugs for this issue:\n\nAffects: fedora-all [bug 1231238]\nAffects: epel-6 [bug 1231239]"
					},
					{
						"isprivate": "0",
						"commentid": "8460256",
						"comment_count": "15",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-07-20 19:47:26 +0000",
						"thetext": "python-pip-7.1.0-1.el7 has been pushed to the Fedora EPEL 7 stable repository.  If problems still persist, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "8460261",
						"comment_count": "16",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-07-20 19:47:43 +0000",
						"thetext": "python-pip-7.1.0-1.el6 has been pushed to the Fedora EPEL 6 stable repository.  If problems still persist, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "8544648",
						"comment_count": "17",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-08-15 02:23:13 +0000",
						"thetext": "bzr-2.6.0-8.fc22 has been pushed to the Fedora 22 stable repository.  If problems still persist, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "8544654",
						"comment_count": "18",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-08-15 02:23:44 +0000",
						"thetext": "bzr-2.6.0-7.fc21 has been pushed to the Fedora 21 stable repository.  If problems still persist, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "9055447",
						"comment_count": "19",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2016-02-12 11:51:56 +0000",
						"thetext": "python-pymongo-2.5.2-8.fc23 has been pushed to the Fedora 23 stable repository. If problems still persist, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "9055541",
						"comment_count": "20",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2016-02-12 12:19:45 +0000",
						"thetext": "python-pymongo-2.5.2-8.fc22 has been pushed to the Fedora 22 stable repository. If problems still persist, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "9080868",
						"comment_count": "21",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2016-02-20 22:58:34 +0000",
						"thetext": "python-pymongo-2.5.2-4.el7 has been pushed to the Fedora EPEL 7 stable repository. If problems still persist, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "9080895",
						"comment_count": "22",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2016-02-20 23:55:27 +0000",
						"thetext": "python-pymongo-2.5.2-3.el6.1 has been pushed to the Fedora EPEL 6 stable repository. If problems still persist, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "9371855",
						"comment_count": "24",
						"who": {
							"text": "errata-xmlrpc",
							"name": "errata-xmlrpc"
						},
						"bug_when": "2016-05-31 10:24:13 +0000",
						"thetext": "This issue has been addressed in the following products:\n\n  Red Hat Software Collections for Red Hat Enterprise Linux 6\n  Red Hat Software Collections for Red Hat Enterprise Linux 7\n  Red Hat Software Collections for Red Hat Enterprise Linux 6.7 EUS\n  Red Hat Software Collections for Red Hat Enterprise Linux 6.6 EUS\n  Red Hat Software Collections for Red Hat Enterprise Linux 7.1 EUS\n  Red Hat Software Collections for Red Hat Enterprise Linux 7.2 EUS\n\nVia RHSA-2016:1166 https://access.redhat.com/errata/RHSA-2016:1166"
					},
					{
						"isprivate": "0",
						"commentid": "10778949",
						"comment_count": "25",
						"who": {
							"text": "anemec",
							"name": "Andrej Nemec"
						},
						"bug_when": "2017-09-08 12:21:34 +0000",
						"thetext": "Statement:\n\nThis issue affects the versions of python27-python-pip, python-pymongo and python-virtualenv as shipped with Red Hat OpenShift 2.x and Satellite 6. Red Hat Product Security has rated this issue as having Low security impact. A future update may address this issue. For additional information, refer to the Issue Severity Classification: https://access.redhat.com/security/updates/classification/."
					}
				]
			}
		},
		{
			"bug_id": 1230678,
			"title": "bzr: use match_hostname() from python-backports-ssl_match_hostname",
			"bugzilla": {
				"bug_id": "1230678",
				"creation_ts": "2015-06-11 10:45:40 +0000",
				"short_desc": "bzr: use match_hostname() from python-backports-ssl_match_hostname",
				"delta_ts": "2015-08-15 02:23:42 +0000",
				"bug_status": "CLOSED",
				"resolution": "ERRATA",
				"priority": "unspecified",
				"bug_severity": "unspecified",
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "8326021",
						"comment_count": "0",
						"who": {
							"text": "thoger",
							"name": "Tomas Hoger"
						},
						"bug_when": "2015-06-11 10:45:40 +0000",
						"thetext": "Description of problem:\n\nbzr contains a copy of match_hostname implementation from Python 3 (in bzrlib/transport/http/_urllib2_wrappers.py).  For Python 2, the function is available via python-backports-ssl_match_hostname.  bzr should depend on that to avoid needing to have fixes for the embedded copy backported.  There were / are few cases when the code needed backport of a security fix - bug 963260, bug 1224999.\n\nVersion-Release number of selected component (if applicable):\n\nbzr-2.6.0-8.fc23"
					},
					{
						"isprivate": "0",
						"commentid": "8402712",
						"comment_count": "1",
						"who": {
							"text": "carl",
							"name": "Carl George"
						},
						"bug_when": "2015-07-02 18:26:10 +0000",
						"thetext": "Eventually python-backports-ssl_match_hostname is going to be removed.  The code from this backport has been fully merged into the standard library in Python 2.7.9.\n\n> The entirety of Python 3.4's ssl module has been backported for Python 2.7.9. See PEP 466 for justification.\n\n* https://www.python.org/downloads/release/python-279/\n* https://www.python.org/dev/peps/pep-0466/\n\nSee also #1229409.\n\nLooking at the source for 2.6.0, patching this should be easy."
					},
					{
						"isprivate": "0",
						"commentid": "8402729",
						"comment_count": "2",
						"who": {
							"text": "carl",
							"name": "Carl George"
						},
						"bug_when": "2015-07-02 18:32:02 +0000",
						"thetext": "Created attachment 1045609\nuse match_hostname function from system ssl module or system backports.ssl_match_hostname module"
					},
					{
						"isprivate": "0",
						"commentid": "8443629",
						"comment_count": "3",
						"who": {
							"text": "jkurik",
							"name": "Jan Kurik"
						},
						"bug_when": "2015-07-15 14:01:22 +0000",
						"thetext": "This bug appears to have been reported against 'rawhide' during the Fedora 23 development cycle.\nChanging version to '23'.\n\n(As we did not run this process for some time, it could affect also pre-Fedora 23 development\ncycle bugs. We are very sorry. It will help us with cleanup during Fedora 23 End Of Life. Thank you.)\n\nMore information and reason for this action is here:\nhttps://fedoraproject.org/wiki/BugZappers/HouseKeeping/Fedora23"
					},
					{
						"isprivate": "0",
						"commentid": "8463126",
						"comment_count": "4",
						"who": {
							"text": "pstodulk",
							"name": "Petr Stodulka"
						},
						"bug_when": "2015-07-21 13:26:14 +0000",
						"thetext": "Created attachment 1054367\nuse match_hostname function from system ssl module or system backports.ssl_match_hostname module\n\nfixed test script"
					},
					{
						"isprivate": "0",
						"commentid": "8464056",
						"comment_count": "5",
						"who": {
							"text": "pstodulk",
							"name": "Petr Stodulka"
						},
						"bug_when": "2015-07-21 16:45:29 +0000",
						"thetext": "Created attachment 1054462\nuse match_hostname function from system ssl module or system backports.ssl_match_hostname module - v3\n\none another wrong import for older python"
					},
					{
						"isprivate": "0",
						"commentid": "8464176",
						"comment_count": "6",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-07-21 17:24:05 +0000",
						"thetext": "bzr-2.6.0-8.fc22 has been submitted as an update for Fedora 22.\nhttps://admin.fedoraproject.org/updates/bzr-2.6.0-8.fc22"
					},
					{
						"isprivate": "0",
						"commentid": "8464179",
						"comment_count": "7",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-07-21 17:26:23 +0000",
						"thetext": "bzr-2.6.0-7.fc21 has been submitted as an update for Fedora 21.\nhttps://admin.fedoraproject.org/updates/bzr-2.6.0-7.fc21"
					},
					{
						"isprivate": "0",
						"commentid": "8488947",
						"comment_count": "8",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-07-29 01:47:39 +0000",
						"thetext": "Package bzr-2.6.0-7.fc21:\n* should fix your issue,\n* was pushed to the Fedora 21 testing repository,\n* should be available at your local mirror within two days.\nUpdate it with:\n# su -c 'yum update --enablerepo=updates-testing bzr-2.6.0-7.fc21'\nas soon as you are able to.\nPlease go to the following url:\nhttps://admin.fedoraproject.org/updates/FEDORA-2015-11995/bzr-2.6.0-7.fc21\nthen log in and leave karma (feedback)."
					},
					{
						"isprivate": "0",
						"commentid": "8544647",
						"comment_count": "9",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-08-15 02:23:11 +0000",
						"thetext": "bzr-2.6.0-8.fc22 has been pushed to the Fedora 22 stable repository.  If problems still persist, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "8544653",
						"comment_count": "10",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-08-15 02:23:42 +0000",
						"thetext": "bzr-2.6.0-7.fc21 has been pushed to the Fedora 21 stable repository.  If problems still persist, please make note of it in this bug report."
					}
				]
			}
		},
		{
			"bug_id": 1230951,
			"security": true,
			"title": "CVE-2013-7440 bzr: python: wildcard matching rules do not follow RFC 6125 [fedora-all]",
			"bugzilla": {
				"bug_id": "1230951",
				"creation_ts": "2015-06-11 21:08:09 +0000",
				"short_desc": "CVE-2013-7440 bzr: python: wildcard matching rules do not follow RFC 6125 [fedora-all]",
				"delta_ts": "2015-08-15 02:23:41 +0000",
				"bug_status": "CLOSED",
				"resolution": "ERRATA",
				"keywords": "Security, SecurityTracking",
				"priority": "medium",
				"bug_severity": "medium",
				"blocked": [
					{
						"bug_id": "1224999",
						"alias": "CVE-2013-7440",
						"creation_ts": "2015-05-26 11:52:59 +0000",
						"short_desc": "CVE-2013-7440 python: wildcard matching rules do not follow RFC 6125",
						"delta_ts": "2021-10-21 00:45:32 +0000",
						"bug_status": "CLOSED",
						"resolution": "ERRATA",
						"keywords": "Security",
						"priority": "low",
						"bug_severity": "low",
						"depends_on": [
							"1023742",
							"1035223",
							"1230951",
							"1230952",
							"1230953",
							"1230954",
							"1231231",
							"1231232",
							"1231238",
							"1231239",
							"1304146",
							"1304225",
							"1304227"
						],
						"blocked": [
							{
								"bug_id": "1225002",
								"error": "NotPermitted"
							}
						],
						"external_bugs": {
							"text": "RHSA-2016:1166",
							"name": "Red Hat Product Errata"
						},
						"long_desc": [
							{
								"isprivate": "0",
								"commentid": "8265872",
								"comment_count": "0",
								"who": {
									"text": "mprpic",
									"name": "Martin Prpiƒç"
								},
								"bug_when": "2015-05-26 11:52:59 +0000",
								"thetext": "It was found that Python's SSL hostname matching rules did not conform to RFC 6125 when the hostname included wildcards.\n\nUpstream issue:\n\nhttps://bugs.python.org/issue17997#msg194950\n\nCVE assignment:\n\nhttp://seclists.org/oss-sec/2015/q2/523\n\nUpstream patch:\n\nhttps://hg.python.org/cpython/rev/10d0edadbcdd"
							},
							{
								"isprivate": "0",
								"commentid": "8325478",
								"comment_count": "1",
								"who": {
									"text": "thoger",
									"name": "Tomas Hoger"
								},
								"bug_when": "2015-06-11 08:18:19 +0000",
								"thetext": "This was fixed upstream in Python 3.3.3, and also in backports.ssl_match_hostname 3.4.0.2:\n\nhttps://pypi.python.org/pypi/backports.ssl_match_hostname/3.4.0.2\n\nCorrected version was also added to Python 2.7.9.  Python 2.7 versions before 2.7.9 did not include match_hostname() and hence were not affected."
							},
							{
								"isprivate": "0",
								"commentid": "8328666",
								"comment_count": "3",
								"who": {
									"text": "thoger",
									"name": "Tomas Hoger"
								},
								"bug_when": "2015-06-11 20:55:07 +0000",
								"thetext": "The backports.ssl_match_hostname is bundled with urllib3 upstream sources.  Additionally, urllib3 is bundled in the requests library.  However, python-urllib3 and python-requests packages in Red Hat products and Fedora do not bundle backports.ssl_match_hostname and instead use the version in python-backports-ssl_match_hostname packages.\n\n\nThere are several multiple copies of match_hostname implementation in python-pip:\n\n- pip/backwardcompat/ssl_match_hostname.py\n  - this was used in pip version prior to 1.5, removed via:\n    https://github.com/pypa/pip/commit/c61b5df\n  - version included in pip 1.4.1 was affected by CVE-2013-7440, but not\n    affected by CVE-2013-2099 (bug 963260)\n  - version included in pip 1.3.4 is affected by both CVE-2013-7440 and\n    CVE-2013-2099, but some Red Hat packages already include patch for\n    CVE-2013-2099\n\n- pip/_vendor/requests/packages/urllib3/packages/ssl_match_hostname/_implementation.py\n  - embedded requests library with embedded urllib3 with embedded\n    backports.ssl_match_hostname\n  - added in pip 1.5 when requests 2.0.0 was added, using ssl_match_hostname\n    version with this issue already fixed\n    https://github.com/pypa/pip/commit/651a961\n\n- pip/_vendor/distlib/compat.py\n  - code was affected by both CVE-2013-2099 and CVE-2013-7440 until pip 6.0.0,\n    when embedded distlib was updated to version 0.2.0:\n    https://github.com/pypa/pip/commit/320a07f\n  - it's unclear to me if match_hostname copy in distlib is used by pip\n\n\ndistlib is also packaged separately in Fedora.  The current version of python-distlib packages is 0.1.9.  CVE-2013-2099 and CVE-2013-7440 were only fixed upstream in 0.2.0 via:\nhttps://bitbucket.org/vinay.sajip/distlib/commits/1e44690\n\n\nmatch_hostname code is also copied in the setuptools package.\n\n- 0.7 introduced the code when it add support for SSL certificate verification\n  https://bitbucket.org/pypa/setuptools/commits/8dc5794\n- 0.9.5 corrected CVE-2013-2099\n  https://bitbucket.org/pypa/setuptools/commits/4b0fb61\n- 1.3 corrected CVE-2013-7440\n  https://bitbucket.org/pypa/setuptools/commits/7eeb678\n\npython-setuptools in Red Hat Enterprise Linux 6 and earlier are based on upstream versions prior to 0.7 and hence do not contain match_hostname.  The version in Red Hat Enterprise Linux 7 is 0.9.8.  However, it does not contain match_hostname, as it was patched to use implementation from the python-backports-ssl_match_hostname package.  Version 0.9.8 is also used in python-setuptools packages in python27 and python33 collections in Red Hat Software Collections.  Those packages already include a patch for CVE-2013-7440.  The rh-python34 collections contains newer upstream setuptools versions (11.3.1)."
							},
							{
								"isprivate": "0",
								"commentid": "8328694",
								"comment_count": "4",
								"who": {
									"text": "thoger",
									"name": "Tomas Hoger"
								},
								"bug_when": "2015-06-11 21:08:40 +0000",
								"thetext": "\nCreated python-pip tracking bugs for this issue:\n\nAffects: fedora-all [bug 1230953]\nAffects: epel-all [bug 1230954]"
							},
							{
								"isprivate": "0",
								"commentid": "8328695",
								"comment_count": "5",
								"who": {
									"text": "thoger",
									"name": "Tomas Hoger"
								},
								"bug_when": "2015-06-11 21:08:48 +0000",
								"thetext": "\nCreated bzr tracking bugs for this issue:\n\nAffects: fedora-all [bug 1230951]"
							},
							{
								"isprivate": "0",
								"commentid": "8328696",
								"comment_count": "6",
								"who": {
									"text": "thoger",
									"name": "Tomas Hoger"
								},
								"bug_when": "2015-06-11 21:08:54 +0000",
								"thetext": "\nCreated python-distlib tracking bugs for this issue:\n\nAffects: fedora-all [bug 1230952]"
							},
							{
								"isprivate": "0",
								"commentid": "8330747",
								"comment_count": "7",
								"who": {
									"text": "thoger",
									"name": "Tomas Hoger"
								},
								"bug_when": "2015-06-12 12:29:18 +0000",
								"thetext": "pip and setuptools are bundled with virtualenv.  The following provides a quick overview of versions bundled with virtualenv versions as found in Red Hat products and Fedora:\n\n- virtualenv 1.7.2 - includes distribute-0.6.27 and setuptools-0.6c11 - not affected\n- virtualenv 1.10.1 - includes pip-1.4.1 and setuptools-0.9.8 - affected\n- virtualenv 1.11.6 - includes pip-1.5.6 and setuptools-3.6 - only pip/distlib affected\n- virtualenv 12.0.7 - includes pip-6.0.8 and setuptools-12.0.5 - not affected"
							},
							{
								"isprivate": "0",
								"commentid": "8330795",
								"comment_count": "8",
								"who": {
									"text": "thoger",
									"name": "Tomas Hoger"
								},
								"bug_when": "2015-06-12 12:45:51 +0000",
								"thetext": "match_hostname is also embedded in pymongo:\n\n- it was introduced in 2.5:\n  https://jira.mongodb.org/browse/PYTHON-466\n  https://github.com/mongodb/mongo-python-driver/commit/48046b2e\n- CVE-2013-2099 was corrected in 2.6:\n  https://jira.mongodb.org/browse/PYTHON-522\n  https://github.com/mongodb/mongo-python-driver/commit/e4f6e4f7\n- CVE-2013-7440 was corrected in 2.7\n  https://jira.mongodb.org/browse/PYTHON-650\n  https://github.com/mongodb/mongo-python-driver/commit/9b0e542a"
							},
							{
								"isprivate": "0",
								"commentid": "8330920",
								"comment_count": "9",
								"who": {
									"text": "thoger",
									"name": "Tomas Hoger"
								},
								"bug_when": "2015-06-12 13:13:39 +0000",
								"thetext": "\nCreated python-pymongo tracking bugs for this issue:\n\nAffects: fedora-all [bug 1231231]\nAffects: epel-all [bug 1231232]"
							},
							{
								"isprivate": "0",
								"commentid": "8330979",
								"comment_count": "10",
								"who": {
									"text": "thoger",
									"name": "Tomas Hoger"
								},
								"bug_when": "2015-06-12 13:23:18 +0000",
								"thetext": "\nCreated zeroinstall-injector tracking bugs for this issue:\n\nAffects: fedora-all [bug 1231238]\nAffects: epel-6 [bug 1231239]"
							},
							{
								"isprivate": "0",
								"commentid": "8460256",
								"comment_count": "15",
								"who": {
									"text": "updates",
									"name": "Fedora Update System"
								},
								"bug_when": "2015-07-20 19:47:26 +0000",
								"thetext": "python-pip-7.1.0-1.el7 has been pushed to the Fedora EPEL 7 stable repository.  If problems still persist, please make note of it in this bug report."
							},
							{
								"isprivate": "0",
								"commentid": "8460261",
								"comment_count": "16",
								"who": {
									"text": "updates",
									"name": "Fedora Update System"
								},
								"bug_when": "2015-07-20 19:47:43 +0000",
								"thetext": "python-pip-7.1.0-1.el6 has been pushed to the Fedora EPEL 6 stable repository.  If problems still persist, please make note of it in this bug report."
							},
							{
								"isprivate": "0",
								"commentid": "8544648",
								"comment_count": "17",
								"who": {
									"text": "updates",
									"name": "Fedora Update System"
								},
								"bug_when": "2015-08-15 02:23:13 +0000",
								"thetext": "bzr-2.6.0-8.fc22 has been pushed to the Fedora 22 stable repository.  If problems still persist, please make note of it in this bug report."
							},
							{
								"isprivate": "0",
								"commentid": "8544654",
								"comment_count": "18",
								"who": {
									"text": "updates",
									"name": "Fedora Update System"
								},
								"bug_when": "2015-08-15 02:23:44 +0000",
								"thetext": "bzr-2.6.0-7.fc21 has been pushed to the Fedora 21 stable repository.  If problems still persist, please make note of it in this bug report."
							},
							{
								"isprivate": "0",
								"commentid": "9055447",
								"comment_count": "19",
								"who": {
									"text": "updates",
									"name": "Fedora Update System"
								},
								"bug_when": "2016-02-12 11:51:56 +0000",
								"thetext": "python-pymongo-2.5.2-8.fc23 has been pushed to the Fedora 23 stable repository. If problems still persist, please make note of it in this bug report."
							},
							{
								"isprivate": "0",
								"commentid": "9055541",
								"comment_count": "20",
								"who": {
									"text": "updates",
									"name": "Fedora Update System"
								},
								"bug_when": "2016-02-12 12:19:45 +0000",
								"thetext": "python-pymongo-2.5.2-8.fc22 has been pushed to the Fedora 22 stable repository. If problems still persist, please make note of it in this bug report."
							},
							{
								"isprivate": "0",
								"commentid": "9080868",
								"comment_count": "21",
								"who": {
									"text": "updates",
									"name": "Fedora Update System"
								},
								"bug_when": "2016-02-20 22:58:34 +0000",
								"thetext": "python-pymongo-2.5.2-4.el7 has been pushed to the Fedora EPEL 7 stable repository. If problems still persist, please make note of it in this bug report."
							},
							{
								"isprivate": "0",
								"commentid": "9080895",
								"comment_count": "22",
								"who": {
									"text": "updates",
									"name": "Fedora Update System"
								},
								"bug_when": "2016-02-20 23:55:27 +0000",
								"thetext": "python-pymongo-2.5.2-3.el6.1 has been pushed to the Fedora EPEL 6 stable repository. If problems still persist, please make note of it in this bug report."
							},
							{
								"isprivate": "0",
								"commentid": "9371855",
								"comment_count": "24",
								"who": {
									"text": "errata-xmlrpc",
									"name": "errata-xmlrpc"
								},
								"bug_when": "2016-05-31 10:24:13 +0000",
								"thetext": "This issue has been addressed in the following products:\n\n  Red Hat Software Collections for Red Hat Enterprise Linux 6\n  Red Hat Software Collections for Red Hat Enterprise Linux 7\n  Red Hat Software Collections for Red Hat Enterprise Linux 6.7 EUS\n  Red Hat Software Collections for Red Hat Enterprise Linux 6.6 EUS\n  Red Hat Software Collections for Red Hat Enterprise Linux 7.1 EUS\n  Red Hat Software Collections for Red Hat Enterprise Linux 7.2 EUS\n\nVia RHSA-2016:1166 https://access.redhat.com/errata/RHSA-2016:1166"
							},
							{
								"isprivate": "0",
								"commentid": "10778949",
								"comment_count": "25",
								"who": {
									"text": "anemec",
									"name": "Andrej Nemec"
								},
								"bug_when": "2017-09-08 12:21:34 +0000",
								"thetext": "Statement:\n\nThis issue affects the versions of python27-python-pip, python-pymongo and python-virtualenv as shipped with Red Hat OpenShift 2.x and Satellite 6. Red Hat Product Security has rated this issue as having Low security impact. A future update may address this issue. For additional information, refer to the Issue Severity Classification: https://access.redhat.com/security/updates/classification/."
							}
						]
					}
				],
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "8328685",
						"comment_count": "0",
						"who": {
							"text": "thoger",
							"name": "Tomas Hoger"
						},
						"bug_when": "2015-06-11 21:08:09 +0000",
						"thetext": "\nThis is an automatically created tracking bug!  It was created to ensure\nthat one or more security vulnerabilities are fixed in affected versions\nof Fedora.\n\nFor comments that are specific to the vulnerability please use bugs filed\nagainst the \"Security Response\" product referenced in the \"Blocks\" field.\n\nFor more information see:\nhttp://fedoraproject.org/wiki/Security/TrackingBugs\n\nWhen submitting as an update, use the fedpkg template provided in the next\ncomment(s).  This will include the bug IDs of this tracking bug as well as\nthe relevant top-level CVE bugs.\n\nPlease also mention the CVE IDs being fixed in the RPM changelog and the\nfedpkg commit message.\n\nNOTE: this issue affects multiple supported versions of Fedora. While only\none tracking bug has been filed, please correct all affected versions at\nthe same time.  If you need to fix the versions independent of each other,\nyou may clone this bug as appropriate.\n\n[bug automatically created by: add-tracking-bugs]"
					},
					{
						"isprivate": "0",
						"commentid": "8328686",
						"comment_count": "1",
						"who": {
							"text": "thoger",
							"name": "Tomas Hoger"
						},
						"bug_when": "2015-06-11 21:08:15 +0000",
						"thetext": "\nUse the following template to for the 'fedpkg update' request to submit an\nupdate for this issue as it contains the top-level parent bug(s) as well as\nthis tracking bug.  This will ensure that all associated bugs get updated\nwhen new packages are pushed to stable.\n\n=====\n\n# bugfix, security, enhancement, newpackage (required)\ntype=security\n\n# testing, stable\nrequest=testing\n\n# Bug numbers: 1234,9876\nbugs=1224999,1230951\n\n# Description of your update\nnotes=Security fix for CVE-2013-7440\n\n# Enable request automation based on the stable/unstable karma thresholds\nautokarma=True\nstable_karma=3\nunstable_karma=-3\n\n# Automatically close bugs when this marked as stable\nclose_bugs=True\n\n# Suggest that users restart after update\nsuggest_reboot=False\n\n======\n\nAdditionally, you may opt to use the bodhi update submission link instead:\n\nhttps://admin.fedoraproject.org/updates/new/?type_=security&bugs=1224999,1230951"
					},
					{
						"isprivate": "0",
						"commentid": "8464175",
						"comment_count": "2",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-07-21 17:24:03 +0000",
						"thetext": "bzr-2.6.0-8.fc22 has been submitted as an update for Fedora 22.\nhttps://admin.fedoraproject.org/updates/bzr-2.6.0-8.fc22"
					},
					{
						"isprivate": "0",
						"commentid": "8464178",
						"comment_count": "3",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-07-21 17:26:21 +0000",
						"thetext": "bzr-2.6.0-7.fc21 has been submitted as an update for Fedora 21.\nhttps://admin.fedoraproject.org/updates/bzr-2.6.0-7.fc21"
					},
					{
						"isprivate": "0",
						"commentid": "8488946",
						"comment_count": "4",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-07-29 01:47:37 +0000",
						"thetext": "Package bzr-2.6.0-7.fc21:\n* should fix your issue,\n* was pushed to the Fedora 21 testing repository,\n* should be available at your local mirror within two days.\nUpdate it with:\n# su -c 'yum update --enablerepo=updates-testing bzr-2.6.0-7.fc21'\nas soon as you are able to.\nPlease go to the following url:\nhttps://admin.fedoraproject.org/updates/FEDORA-2015-11995/bzr-2.6.0-7.fc21\nthen log in and leave karma (feedback)."
					},
					{
						"isprivate": "0",
						"commentid": "8544646",
						"comment_count": "5",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-08-15 02:23:03 +0000",
						"thetext": "bzr-2.6.0-8.fc22 has been pushed to the Fedora 22 stable repository.  If problems still persist, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "8544652",
						"comment_count": "6",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2015-08-15 02:23:41 +0000",
						"thetext": "bzr-2.6.0-7.fc21 has been pushed to the Fedora 21 stable repository.  If problems still persist, please make note of it in this bug report."
					}
				]
			}
		}
	],
	"builds": [
		{
			"nvr": "bzr-2.6.0-7.fc21",
			"signed": true,
			"type": "rpm",
			"package": {
				"armv7hl": [
					{
						"name": "bzr",
						"version": "2.6.0",
						"release": "7.fc21",
						"arch": "armv7hl"
					},
					{
						"name": "bzr-debuginfo",
						"version": "2.6.0",
						"release": "7.fc21",
						"arch": "armv7hl"
					}
				],
				"i686": [
					{
						"name": "bzr-debuginfo",
						"version": "2.6.0",
						"release": "7.fc21",
						"arch": "i686"
					},
					{
						"name": "bzr",
						"version": "2.6.0",
						"release": "7.fc21",
						"arch": "i686"
					}
				],
				"noarch": [
					{
						"name": "bzr-doc",
						"version": "2.6.0",
						"release": "7.fc21",
						"arch": "noarch"
					}
				],
				"src": [
					{
						"name": "bzr",
						"version": "2.6.0",
						"release": "7.fc21",
						"arch": "src"
					}
				],
				"x86_64": [
					{
						"name": "bzr",
						"version": "2.6.0",
						"release": "7.fc21",
						"arch": "x86_64"
					},
					{
						"name": "bzr-debuginfo",
						"version": "2.6.0",
						"release": "7.fc21",
						"arch": "x86_64"
					}
				]
			}
		}
	],
	"close_bugs": true,
	"comments": [
		{
			"id": 85256,
			"text": "This update has been submitted for testing by pstodulk. ",
			"timestamp": "2015-07-21 17:26:26",
			"update_id": 11793,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 85257,
			"text": "Taskotron: depcheck test PASSED on x86_64. Result log:\nhttps://taskotron.fedoraproject.org/taskmaster//builders/x86_64/builds/108257/steps/runtask/logs/stdio\n(results are informative only)",
			"timestamp": "2015-07-21 17:29:48",
			"update_id": 11793,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/8cba34b3793397cca75782c70d6a4c3af6fc4c8e7d502fad36609e3b4b4670f9?s=24&d=retro",
				"email": "taskotron@fedoraproject.org",
				"groups": [
					{}
				],
				"id": 100,
				"name": "taskotron",
				"openid": "taskotron.id.fedoraproject.org"
			},
			"user_id": 100
		},
		{
			"id": 85258,
			"text": "Taskotron: depcheck test PASSED on i386. Result log:\nhttps://taskotron.fedoraproject.org/taskmaster//builders/x86_64/builds/108263/steps/runtask/logs/stdio\n(results are informative only)",
			"timestamp": "2015-07-21 17:44:00",
			"update_id": 11793,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/8cba34b3793397cca75782c70d6a4c3af6fc4c8e7d502fad36609e3b4b4670f9?s=24&d=retro",
				"email": "taskotron@fedoraproject.org",
				"groups": [
					{}
				],
				"id": 100,
				"name": "taskotron",
				"openid": "taskotron.id.fedoraproject.org"
			},
			"user_id": 100
		},
		{
			"id": 85259,
			"text": "This update is currently being pushed to the Fedora 21 testing updates repository.",
			"timestamp": "2015-07-23 18:58:32",
			"update_id": 11793,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 85260,
			"text": "This update is currently being pushed to the Fedora 21 testing updates repository.",
			"timestamp": "2015-07-23 19:05:38",
			"update_id": 11793,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 85261,
			"text": "This update has been pushed to testing",
			"timestamp": "2015-07-29 01:47:38",
			"update_id": 11793,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 85262,
			"karma": 1,
			"text": "no regressios noted",
			"timestamp": "2015-07-29 22:45:01",
			"update_id": 11793,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/f4394b8fb4c9a99c4b534dc724912fe75a5b87fe15048985ece8388c2441d0ca?s=24&d=retro",
				"email": "rosset.filipe@gmail.com",
				"groups": [
					{
						"name": "proventesters"
					},
					{
						"name": "provenpackager"
					},
					{
						"name": "packager"
					},
					{
						"name": "trust admins"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "fedora-contributor"
					},
					{
						"name": "l10n"
					},
					{
						"name": "cvsl10n"
					},
					{
						"name": "ipausers"
					},
					{
						"name": "fedora-br"
					},
					{
						"name": "fedorabugs"
					},
					{
						"name": "ambassadors"
					}
				],
				"id": 124,
				"name": "filiperosset",
				"openid": "filiperosset.id.fedoraproject.org"
			},
			"user_id": 124
		},
		{
			"id": 85263,
			"text": "This update has reached 7 days in testing and can be pushed to stable now if the maintainer wishes",
			"timestamp": "2015-08-05 06:32:56",
			"update_id": 11793,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 85264,
			"text": "This update has been submitted for stable by pstodulk. ",
			"timestamp": "2015-08-13 10:58:04",
			"update_id": 11793,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 85265,
			"text": "Taskotron: upgradepath test PASSED on noarch. Result log:\nhttps://taskotron.fedoraproject.org/taskmaster//builders/x86_64/builds/116196/steps/runtask/logs/stdio\n(results are informative only)",
			"timestamp": "2015-08-13 10:59:29",
			"update_id": 11793,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/8cba34b3793397cca75782c70d6a4c3af6fc4c8e7d502fad36609e3b4b4670f9?s=24&d=retro",
				"email": "taskotron@fedoraproject.org",
				"groups": [
					{}
				],
				"id": 100,
				"name": "taskotron",
				"openid": "taskotron.id.fedoraproject.org"
			},
			"user_id": 100
		},
		{
			"id": 85266,
			"text": "This update is currently being pushed to the Fedora 21 stable updates repository.",
			"timestamp": "2015-08-13 21:46:24",
			"update_id": 11793,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 85267,
			"text": "This update is currently being pushed to the Fedora 21 stable updates repository.",
			"timestamp": "2015-08-14 14:18:59",
			"update_id": 11793,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 85268,
			"text": "This update has been pushed to stable",
			"timestamp": "2015-08-15 02:23:48",
			"update_id": 11793,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		}
	],
	"content_type": "rpm",
	"date_pushed": "2015-08-15 02:23:48",
	"date_stable": "2015-08-15 02:23:48",
	"date_submitted": "2015-07-21 17:24:10",
	"date_testing": "2015-07-29 01:47:38",
	"karma": 1,
	"notes": "Security fix for CVE-2013-7440",
	"pushed": true,
	"release": {
		"branch": "f21",
		"candidate_tag": "f21-updates-candidate",
		"composed_by_bodhi": true,
		"dist_tag": "f21",
		"id_prefix": "FEDORA",
		"long_name": "Fedora 21",
		"mail_template": "fedora_errata_template",
		"name": "F21",
		"override_tag": "f21-override",
		"package_manager": "dnf",
		"pending_stable_tag": "f21-updates-pending",
		"pending_testing_tag": "f21-updates-testing-pending",
		"stable_tag": "f21-updates",
		"state": "archived",
		"testing_repository": "updates-testing",
		"testing_tag": "f21-updates-testing",
		"version": "21"
	},
	"severity": "unspecified",
	"stable_karma": 3,
	"status": "stable",
	"suggest": "unspecified",
	"title": "bzr-2.6.0-7.fc21",
	"type": "security",
	"url": "https://bodhi.fedoraproject.org/updates/FEDORA-2015-11995",
	"unstable_karma": -3,
	"updateid": "FEDORA-2015-11995",
	"user": {
		"avatar": "https://seccdn.libravatar.org/avatar/79b5c3f14716ad77643e3da54f4ebb9e75c4b9a01aadad280491186d73da3943?s=24&d=retro",
		"email": "pstodulk@redhat.com",
		"groups": [
			{
				"name": "packager"
			},
			{
				"name": "ipausers"
			},
			{
				"name": "fedora-contributor"
			},
			{
				"name": "signed_fpca"
			},
			{
				"name": "fedorabugs"
			},
			{
				"name": "git-maint"
			},
			{
				"name": "oamg"
			},
			{
				"name": "leapp"
			}
		],
		"id": 890,
		"name": "pstodulk",
		"openid": "pstodulk.id.fedoraproject.org"
	},
	"version_hash": "2570c154cdbb1fa260c72fea7809b26aec937b3a"
}
